locales/en.json

```
@@ -6,7 +6,7 @@
     "admin_password_changed": "The administration password got changed",
     "admin_password_too_long": "Please choose a password shorter than 127 characters",
     "already_up_to_date": "Nothing to do. Everything is already up-to-date.",
-    "app_action_cannot_be_ran_because_required_services_down": "This app requires some services which are currently down. Before continuing, you should try to restart the following services (and possibly investigate why they are down): {services}",
+    "app_action_cannot_be_ran_because_required_services_down": "This app requires more services to run. Before continuing, try to restart the following services (and possibly investigate why they are down): {services}",
```

To me this "more" is confusing, it sounds like you need a quantitative "X" services running whereas the emphasis should be more on *some specific services* should be running...

```suggestion
    "app_action_cannot_be_ran_because_required_services_down": "To perform this action, some specific services should to be running. Before continuing, try to restart the following services (and possibly investigate why they are down): {services}",
```